tests/operator/operator.go

```diff
@@ -247,7 +247,18 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 					Expect(err).ToNot(HaveOccurred())
 				}
 
-				vmi := libvmifact.NewCirros(
+				tests.CreateConfigMap(configMapName, testsuite.GetTestNamespace(nil), config_data)
+				tests.CreateSecret(secretName, testsuite.GetTestNamespace(nil), secret_data)
```

Comment:
Aren't we suppose to get rid of this function?

Reply:
sorry I miss this